src/kOS/Suffixed/Addons.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using kOS.Safe.Encapsulation.Suffixes;
+using kOS.Safe.Encapsulation;
+
+namespace kOS.Suffixed
+{
+    public class AddonList : Structure
+    {
+        private Addon KACAddon;

```

Comment:
in this case we could simply not have these fields and lazy new them when the suffix is called, which i prefer anyway. the conflict would get to remain unresolved another day ;)